.devcontainer/postCreateCommand.sh

```diff
@@ -1,7 +1,20 @@
 #!/bin/bash
 
-sudo apt-get update
-sudo apt-get install -y libgraphviz-dev
+sudo apt-get install -y libgraphviz-dev git-lfs
 
-# Install dev packages.
-pip3 install python/mlcroissant/.[dev]
+# Install dependencies except mlcroissant itself
+pip install absl-py \
```

Comment:
Not possible to do something like `pip install .[dev]`? (No idea, would just be easier to maintain)